feature/amazon/src/test/java/com/revenuecat/purchases/amazon/ProxyAmazonBillingActivityBroadcastReceiverTest.kt

```diff
@@ -0,0 +1,47 @@
+package com.revenuecat.purchases.amazon;
+
+import android.app.Activity
+import android.content.Intent
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import io.mockk.clearAllMocks
+import io.mockk.every
+import io.mockk.just
+import io.mockk.mockk
+import io.mockk.runs
+import io.mockk.verify
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class ProxyAmazonBillingActivityBroadcastReceiverTest {
+
+    private lateinit var underTest: ProxyAmazonBillingActivityBroadcastReceiver
+    private lateinit var mockActivity: Activity
+
+    @Before
+    fun setup() {
+        mockActivity = mockk()
+        every {
+            mockActivity.finish()
+        } just runs
+        underTest = ProxyAmazonBillingActivityBroadcastReceiver(mockActivity)
+    }
+
+    @After
+    fun tearDown() {
+        clearAllMocks()
```

Comment:
NABD but currently we are reseting all mocks on the `@Before` block, so this isn't really needed. If we created the mocks and didn't create new ones for each test, then this would be needed.

Reply:
Oh yes, true, I will remove it